### PR TITLE
[Feature] Slack message timeout bypass

### PR DESCRIPTION
**Pull request by Sqyphen**
## Description

Update to send confirmation message to user, even after the Slack modal 3 second timeout

## Implementation

Updated the confirmation messages to be sent via `request_url` instead of updating the modal view.

## Screenshot
n/a

## .env changes
n/a

## Additional requirements
n/a

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -27,51 +27,92 @@ app.use(bodyParser.json());
...
+		headers: {
+			'Content-Type': 'application/json',
+			Authorization: 'Bearer ' + process.env.SLACK_TOKEN
+		}
+	};
+
+	return fetch(url, options)
+		.then(response => {
+			if(response.error != null ){
+				console.log('error')
```
Comment (Lily2point0) [inline index.js]: you should throw an error so the `catch` can get it (and handle it in the catch eventually...)

File: index.js [hunk 2]
```diff
@@ -27,51 +27,92 @@ app.use(bodyParser.json());
...
+		.catch(err => console.log(err));
+}
+
 async function setUpStructure() {
 	await Structure.init();
 }
 
-app.listen(PORT, () => console.log(`Example app listening on port ${PORT}!`));
\ No newline at end of file
+function wait(ms) {
```
Comment (Lily2point0) [inline index.js]: Might be worth moving to a util file, especially as it is for dev only

File: index.js (hunk 1 above)
Comment (Sqyphen) [reply index.js]: Done

File: index.js (hunk 2 above)
Comment (Sqyphen) [reply index.js]: Done

**Pull request merged**